pkgs/top-level/perl-packages.nix

```diff
@@ -13472,6 +13472,30 @@ let
     };
   };
 
+  Shelldap = buildPerlPackage rec {
+    name = "shelldap-${version}";
+    version = "1.4.0";
+    src = fetchurl {
+      url = "https://bitbucket.org/mahlon/shelldap/downloads/shelldap-${version}.tar.gz";
+      sha256 = "07gkvvxcgw3pgkfy8p9mmidakciaq1rsq5zhmdqd8zcwgqkrr24i";
+    };
+    buildInputs = [ perl YAMLSyck NetLDAP AlgorithmDiff IOSocketSSL AuthenSASL TermReadLineGnu TermShell ];
+    prePatch = ''
+      touch Makefile.PL
+    '';
+    installPhase = ''
+      mkdir -p $out/bin
+      cp shelldap $out/bin
+    '';
+    outputs = [ "out" ];
```

Comment:
you don't need this one as it defaults to `out`.

Reply:
True. I'll correct that.

Reply:
Hum wait... When I remove it I get an error:

    builder for '/nix/store/d76y3x4k9bj6ffgpmlqddb47kvdl8jgc-perl5.28.1-shelldap-1.4.0.drv' failed to produce output path '/nix/store/2jflsx7yf73c4r820dbv91hmn2zgpswr-perl5.28.1-shelldap-1.4.0-devdoc'
    error: build of '/nix/store/d76y3x4k9bj6ffgpmlqddb47kvdl8jgc-perl5.28.1-shelldap-1.4.0.drv' failed

Maybe the `perlPackage` changes this default?

Reply:
hmmm, interesting. Maybe @volth can shed more light on this.

Reply:
yes, ```buildPerlPackage``` sets ```output = ["out" "devdoc"]```, so one need to write ```output = ["out"]``` it there is only "out"